vtex-apps/order-details
# Feature/tlva 216/orders link

#### What does this PR do? \*
Update link de seccion order de mi cuenta de la pagina order-confirmation

#### How to test it? \*
[workspace](https://tlva216--thefoschini.myvtex.com/)
https://www.loom.com/share/bc793b5f76c048fba0da52f398eef8b5

 - Place an order
 - on the order-confirmation page, click on the "Update Order" button.
 - The next page should be: https://tlva216--thefoschini.myvtex.com/account#/tfg-orders/{id-order}

You can also try my account

Credentials:
user: jesus.rodrigo@acct.global pass: Acct12345
URL: https://tlva216--thefoschini.myvtex.com/checkout/orderPlaced/?og=1234641301303

#### Related to / Depends on \*
